scripts/bundles/sys-node.ts

```diff
@@ -4,7 +4,7 @@ import fs from 'fs-extra';
 import { join } from 'path';
 import type { RollupOptions } from 'rollup';
 import { minify } from 'terser';
-import webpack from 'webpack';
+import webpack, { Configuration } from 'webpack';
```

Comment:
I recommend hiding whitespace for this file - by extracting the config to it's own variable, that threw off some whitespace indentation